samples/msgext-northwind-inventory-csharp/Program.cs

```diff
@@ -0,0 +1,83 @@
+using msgext_northwind_inventory_csharp;
+using msgext_northwind_inventory_csharp.Bots;
+using msgext_northwind_inventory_csharp.DbSetup;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Bot.Builder.BotFramework;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using msgext_northwind_inventory_csharp.NorthwindDB;
```

Comment:
Remove unnecessary usings

```suggestion
using msgext_northwind_inventory_csharp;
using msgext_northwind_inventory_csharp.Bots;
using msgext_northwind_inventory_csharp.DbSetup;
using Microsoft.Bot.Builder;
using Microsoft.Bot.Builder.Integration.AspNet.Core;
using Microsoft.Bot.Connector.Authentication;
```

Reply:
fixed